vsintegration/tests/unittests/Tests.LanguageService.Colorizer.fs

```diff
@@ -1,1091 +0,0 @@
-// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
@otawfik-ms I'm sure you're right. I suppose I just wanted to be able to review the changes in the tests, to see if anything has been lost inadvertently (or to understand why if on purpose)